spring-cloud-stream-app-starters-docs/pom.xml

```diff
@@ -196,7 +196,7 @@
 								</goals>
 								<configuration>
 									<target>
-										<zip destfile="${project.build.directory}/spring-cloud-stream-modules-${project.version}.zip">
+										<zip destfile="${project.build.directory}/spring-cloud-stream-app-starters-docs-${project.version}.jar">
```

Comment:
looks to have already been merged and using artifactId instead of hardcoding it... :)